x/retry/policyprovider.go

```diff
@@ -84,5 +90,8 @@ func (ppp *ProcedurePolicyProvider) Policy(_ context.Context, req *transport.Req
 	if pol, ok := ppp.serviceProcedureToPolicy[serviceProcedure{Service: req.Service}]; ok {
 		return pol
 	}
+	if pol, ok := ppp.serviceProcedureToPolicy[serviceProcedure{Procedure: req.Procedure}]; ok {
+		return pol
+	}
```

Comment:
Hmm, I'm not 100% sure tbh, this will depend on how people use the retry middleware.